src/Sentry/SentryPropagationContext.cs

```diff
@@ -33,13 +29,19 @@ internal class SentryPropagationContext
         TraceId = traceId;
         SpanId = SpanId.Create();
         ParentSpanId = parentSpanId;
-        DynamicSamplingContext = dynamicSamplingContext;
+        _dynamicSamplingContext = dynamicSamplingContext;
     }
 
-    public SentryPropagationContext()
+    public SentryPropagationContext() : this(null)
+    { }
```

Comment:
When this constructor gets called in the `CreateFromHeaders` method, won't the TraceId and SpanId be empty? Is there ever a scenario where we'd want that to be the case? 

Even if we don't get it from upstream, I think we always want a TraceId to link this span to downstream events right?